### DISCUSSION TITLE
Please add V2H Feature to card

### DISCUSSION THREAD

**Discussion by terzo33** [slipx06/sunsynk-power-flow-card#544] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I can't see V2H (vehicle to home) icon and sensors (hz,V,A). Actually I'm used Grid Power for V2H but there is no way to see how much power the two power inputs are sending to the inverter. the only possibility at the moment is to choose V2H or grid power :(

### Expected behaviour

Please add V2H (vehicle to home) icon and sensors (Hz,V,A) near Grid Power icon

### Mode

lite

### Context / Reason

I have ATS for switch automatically V2H and Grid Power.
Please add V2H (vehicle to home) icon and sensors (Hz,V,A) near Grid Power icon.
I see the V2H values ​​via a smart energy sensor

**Comment by slipx06** [comment]
HI. Have you tried to use the AUX options to show this? 

![image](https://github.com/user-attachments/assets/9c776d7f-317c-45a8-aaa6-4cbb33d48773)

**Comment by terzo33** [comment]
> HI. Have you tried to use the AUX options to show this? 
> 
> ![image](https://github.com/user-attachments/assets/9c776d7f-317c-45a8-aaa6-4cbb33d48773)
> 

Hi @slipx06 , aux option is for load (inverter Output), V2L or V2H is like a grid charge(energy to inverter)

put an icon for replace "non-essential" icon and values with the icon of the machine and values with the energy going to the inverter


Thanks

**Comment by slipx06** [comment]
You can use the AUX port to send energy to the inverter. Some people connect a generator or another micro-inverter. Or you can also use the non-essential circuit. 

This is how some other users have configured it

![image](https://github.com/user-attachments/assets/87435da6-2756-4d69-89c5-a84da979c115)

https://community.home-assistant.io/t/inverter-power-flow-card/562933/438

**Comment by terzo33** [comment]
Thanks a lot. I will try but maybe with aux option is can't see voltage and frequency values
